docs/src/modules/components/ApiPage/sections/ToggleDisplayOption.tsx

```diff
@@ -1,15 +1,15 @@
 import * as React from 'react';
-import Tooltip, { TooltipProps } from '@mui/material/Tooltip';
-import ToggleButtonGroup from '@mui/material/ToggleButtonGroup';
-import ToggleButton, { ToggleButtonProps } from '@mui/material/ToggleButton';
-import CalendarViewDayRoundedIcon from '@mui/icons-material/CalendarViewDayRounded';
-import TableChartRoundedIcon from '@mui/icons-material/TableChartRounded';
-import ReorderRoundedIcon from '@mui/icons-material/ReorderRounded';
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import Menu from '@mui/material/Menu';
+import MenuItem from '@mui/material/MenuItem';
+import CheckIcon from '@mui/icons-material/Check';
+import ArrowDropDownRoundedIcon from '@mui/icons-material/ArrowDropDownRounded';
 import useEnhancedEffect from '@mui/utils/useEnhancedEffect';
 
-export type ApiDisplayOptions = 'collapsed' | 'expanded' | 'table';
+export type ApiDisplayOptions = 'Collapsed' | 'Expanded' | 'Table';
 
-const options: ApiDisplayOptions[] = ['collapsed', 'expanded', 'table'];
+const options: ApiDisplayOptions[] = ['Collapsed', 'Expanded', 'Table'];
```

Comment:
I'm not in favor of touching those.

Users have in their locale storage a saved value `propsLayout = 'table`. If you modify those variables, their saved setting will be lost.

I assume it's because to get the "view: Table" instead of "view: table". for this usecase, you can use ``t(`api-docs.layout.${displayOption}`)``

and in https://github.com/mui/material-ui/blob/master/docs/translations/translations.json add `"layout": { "table": "Table", "expanded": "Expanded", ...}`

Bonus point for this solution, it's future proof for translation

Reply:
Ended up not relying on any translations change given the `textTranform: 'capitalize'` property on the Button was already solving the problem 😬